chart/templates/deployment.yaml

```diff
@@ -50,14 +50,14 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: "{{ .Values.server.servlet.contextPath }}actuator/health"
+              path: "{{ .Values.server.servlet.contextPath }}/actuator/health"
```

Comment:
Does this work when running Kafdrop on the context path `/`? I think last time I tried, it didn't work, as it introduces two slashes.

Reply:
Hi @ekoutanov context path with a slash at the end throws error `IllegalArgumentException` in `org.springframework.boot.web.servlet.server.AbstractServletWebServerFactory.checkContextPath`. See full stacktrace above.

A better default for contextPath would be "". This works. I'll updated the PR

Reply:
Okay updated now. 
Tested and the following contextPath values work now `/` and `/kafkdrop` and `/kafdrop/`

Reply:
I'll give it a crack